tests/examples/marked_tracks.py

```diff
@@ -36,11 +35,18 @@ def test_mark_track3(self, es_cluster, rally):
 
 @pytest.mark.track("test-track")
 class TestMarkedClass:
-    @pytest.mark.track("test-track","test-track2")
     def test_mark_track(self, es_cluster, rally):
-        rally.race(track="test-track",challenge="index-only")
-        rally.race(track="test-track2",challenge="force-merge")
+        rally.race(track="test-track",challenge="force-merge")
 
     @pytest.mark.track("test-track3")
     def test_mark_track3(self, es_cluster, rally):
         rally.race(track="test-track3",challenge="index-only")
+
+@pytest.mark.track("many-tracks")
```

Comment:
I think this marker is incorrect. The track is either `many-tracks/sub-track` or `many-tracks/sub-track2`. Let's assume that track marker points at a, well... track :)

Reply:
What I wanted to express above is we can simplify our lives assuming markers contain full track names as if provided by track fixture. Otherwise we have to handle both short and full track names from `--track-filter` as well as short and full track names from markers which I find too complex for my simple mind.